code/drasil-example/dblpend/lib/Drasil/DblPend/Body.hs

```diff
@@ -215,22 +215,21 @@ sysCtxIntro prog = foldlSP
 sysCtxDesc :: Contents
 sysCtxDesc = foldlSPCol [S "The interaction between the", phraseNP (product_
    `andThe` user), S "is through an application programming" +:+.
-   phrase interface, S "The responsibilities of the", phraseNP (user 
+   phrase interface, S "The responsibilities" `S.ofThe` phraseNP (user 
    `andThe` system), S "are as follows"]
 
 sysCtxUsrResp :: CI -> [Sentence]
 sysCtxUsrResp prog = [S "Provide initial" +:+ pluralNP (condition `ofThePS`
-  physical) +:+ S "state of the" +:+ phrase motion +:+ S "and the" +:+ plural inDatum +:+ S "related to the" +:+
-  phrase prog `sC` S "ensuring no errors in the" +:+
-  plural datum +:+. S "entry",
-  S "Ensure that consistent units are used for" +:+. pluralNP (combineNINI input_ Doc.variable),
+  physical) +:+ S "state" `S.ofThe` phrase motion `S.andThe` plural inDatum +:+ S "related" +:+
```

Comment:
If `state` was a phrase, this split would have been useful. This is not. The `S.andThe` is fine.